addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -302,12 +308,20 @@ options.registry.gallery = options.Class.extend({
      */
     notify: function (name, data) {
         this._super(...arguments);
+        if (!this.hasAddImages) {
+            return;
+        }
```

Comment:
I would add comment here to explain that the widget is instantiate multiple times and we don't want to move the image multiple times, and also that it will be changed in master :+1: